rllib/core/rl_module/marl_module.py

```diff
@@ -231,34 +231,45 @@ def _forward_exploration(
         return self._run_forward_pass("forward_exploration", batch, **kwargs)
 
     @override(RLModule)
-    def get_state(self) -> Mapping[str, Any]:
+    def get_state(self, module_ids: Optional[Set[str]] = None) -> Mapping[str, Any]:
```

Comment:
All these mappings in this file are str->Any, but should by ModuleID->Any, as described in the docstrings.

Reply:
fixed